docs/contents/dev-docs/setup.md

```diff
@@ -11,21 +11,95 @@ the steps below.
+## Prerequisites
 
-First, fork the [upstream source repository](https://github.com/aws/aws-controllers-k8s) 
-into your personal GitHub account. Then, in `$GOPATH/src/github.com/aws/`,
-clone your repo and add the upstream like so:
+Please ensure that you have [properly installed Go][install-go].
 
-```
-git clone git@github.com:$GITHUB_ID/aws-controllers-k8s && \
-cd aws-controllers-k8s && \
-git remote add upstream git@github.com:aws/aws-controllers-k8s
-```
+[install-go]: https://golang.org/doc/install
 
 !!! note "Go version"
     We recommend to use a Go version of `1.14` or above for development.
 
+## Fork upstream repositories
+
+The first step in setting up your ACK development environment is to fork the
+upstream ACK source code repositories to your personal Github account.
+
+There are two common upstream repositories you should fork first:
+
+* `github.com/aws/aws-controllers-k8s` is the core ACK runtime and types
+* `github.com/aws-controllers-k8s/code-generator` is the ACK code generator
+
+!!! tip "prefix ACK forked repos with 'ack-'"
+    When I fork repositories to my personal Github account, I tend to prefix
+    the repositories with a common string for grouping purposes. For ACK source
+    repositories that I forked from the `github.com/aws-controllers-k8s` Github
+    Organization, I prefix those repositories with "ack-". For example, when I
+    orked the `github.com/aws-controllers-k8s/code-generator` repository to my
+    github.com/jaypipes` personal space on Github, I immediately renamed the
+    orked repo to `github.com/jaypipes/ack-code-generator`. This makes it easier
+    o quickly filter repositories that are forked from the
+    github.com/aws-controllers-k8s` Github Organization.
+
+After forking the above common repositories, fork the upstream service
+controller repositories that you wish to work on or test out. The upstream
+service controller repositories are in the `github.com/aws-controllers-k8s`
+Github Organization and follow a naming schema of `$SERVICE_ALIAS-controller`.
+
+So, if you wanted to work on the S3 service controller, you would fork the
+`github.com/aws-controllers-k8s/s3-controller` source repository to your
+personal Github space.
+
+## Ensure source code organization directories exist
+
+Make sure in your `$GOPATH/src` that you have directories for each of the `aws`
+and `aws-controllers-k8s` organizations:
+
+```bash
+mkdir -p $GOPATH/src/github.com/aws
+mkdir -p $GOPATH/src/github.com/aws-controllers-k8s
+```
+
+## `git clone` forked repositories and add upstream remote
+
+For each of your forked repositories, you will `git clone` the repository into
+the appropriate folder in your `$GOPATH`. Once `git clone`'d, you will want to
+set up a Git remote called "upstream" (remember that "origin" will be pointing
+at your forked repository location in your personal Github space).
+
+You can use this script to do this for you:
+
+```bash
+GITHUB_ID="your GH userid"
+
+cd $GOPATH/src/aws
```

Comment:
path incorrect ? should be $GOPATH/src/`github.com`/aws

Reply:
fixed.